en_US.json

```diff
@@ -2303,6 +2304,104 @@
 	"taskListing.tab.deploymentTask": "Deployment",
 	"taskListing.tab.validationTask": "Validation",
 	"taskListing.validation.fetch.error": "An error occurred while fetching validation tasks.",
+	"testsuites.button.back": "Back",
+	"testsuites.button.cancel": "Cancel",
+	"testsuites.button.create.test.case": "Create Test Case",
+	"testsuites.button.delete": "Delete",
+	"testsuites.button.edit": "Edit",
+	"testsuites.button.reset": "Reset",
+	"testsuites.button.run": "Run",
+	"testsuites.button.save": "Save",
+	"testsuites.button.save.and.create.case": "Save and Create Test Case",
+	"testsuites.button.search": "Search",
+	"testsuites.button.search.with.dataindex": "Search {dataIndex}",
+	"testsuites.button.yes.delete": "Yes, delete",
+	"testsuites.confirm.delete.multi": "Delete all({cntTestCase}) test cases?",
+	"testsuites.confirm.text.delete.test.case": "Delete the test case '{testCaseName}' ?",
+	"testsuites.confirm.text.delete.test.suite.name": "Delete the test suite '{testSuiteName}' ?",
+	"testsuites.confirm.text.delete.test.suite.name.more.one.test.cases": "Delete test suite '{testSuiteName}' and its {cntTestCases} test cases?",
+	"testsuites.confirm.text.delete.test.suite.name.only.one.test.cases": "Delete test suite '{testSuiteName}' and its {cntTestCases} test case?",
+	"testsuites.content.discard.case.create": "Discard the test case create?",
```

Comment:
"create" should be "creation".

"create" is a verb only. We need the noun creation in this context.
This differs from "update" which can be used as both a verb and a noun.